llvm/tools/sycl-post-link/sycl-post-link.cpp

```diff
@@ -234,6 +234,12 @@ cl::opt<bool> EmitOnlyKernelsAsEntryPoints{
              "device code split"),
     cl::cat(PostLinkCat), cl::init(false)};
 
+cl::opt<bool> SplitForRunTimeLinking{
+    "split-for-run-time-linking",
```

Comment:
`for` is a bit confusing for me, maybe we can name this `link-external-functions-at-runtime` or something like that?

Reply:
How about `include-external-functions` or `include-exernal-functions-in-device-split`?  The option and variable names should be very similar.

Reply:
sounds good to me!

Reply:
Since this is a new option, let's invert the sense so that it is not required to get previous behavior, `exclude-external-functions`.

Reply:
no strong opinion there

Reply:
Changed option to -exclude-external-functions